### DISCUSSION TITLE
Taking so much time for saving a file - Vscode

### DISCUSSION THREAD

**Discussion by naseef0** [vercel/next.js#50273] [App Router]
When running with  `npm run dev`, after around 30min, Vscode takes so much time for saving a file.
The same issue happened with another laptop also.

![image](https://github.com/vercel/next.js/assets/45784156/66ed598a-fa36-4334-b3bc-98c83afc7cd3)

**Comment by ciruz** [comment]
This could be any vscode extension, since you probably sync your settings and extensions to your laptop aswell, both devices will be affected by the same issue.

Please launch vscode without any extensions from your terminal, to make sure it's not an extension issue, with this command:

`code --disable-extensions`
